ECMA2Yaml/ECMAHelper/ECMALoader.cs

```diff
@@ -455,6 +440,38 @@ private Member LoadMember(Models.Type t, XElement mElement)
             return m;
         }
 
+        public static ReturnValue MonikerizeReturnValue(XElement returnValueElement)
+        {
+            var returnTypes = returnValueElement?.Elements("ReturnType").Select(r => new { ReturnType = r.Value, Monikers = r.Attribute("FrameworkAlternate")?.Value });
+            if (returnTypes == null || !returnTypes.Any()) return null;
+
+            Func<string, HashSet<string>> convertMonikers = fxlist =>
+            {
+                if (string.IsNullOrWhiteSpace(fxlist))
+                    return new HashSet<string>();
+
+                return new HashSet<string>(fxlist.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries).Distinct());
```

Comment:
we can directly use `ECMALoader.LoadFrameworkAlternate()` to parse framework alternate values.

Reply:
Ahh good tip, I've simplified the code to use this